data/core/units/goblins/Knight.cfg

```diff
@@ -15,9 +15,9 @@
     advances_to=Direwolf Rider
     cost=32
     usage=scout
-    description= _ "The Goblin ‘Knights’ have little in common with the men who share that title, and the title is bestowed in mockery by their enemies. The similarity is simply that they are the elite of the wolf riders, likely promoted to their position by their success in some audacious raid.
+    description= _ "Battle experience strengthens not only the physical aspects of mount and rider, but also builds on the bond between them. The term ‘goblin knight’ refers not solely to the rider, but to goblin and wolf together. While a human knight may ride many different steeds in to battle, the pairing of a goblin knight is based on a fierce loyalty between a tamed wild beast, and the person who tamed it.
 
-The wolves they are given are bred for speed and strength, making them quite dangerous in combat."
+Among orcs, the riders are treated as a class above goblins, even when separated from their mount. Orcish grunts, while feeling themselves superior to goblins, recognize that picking a fight with the rider is also picking a fight with jaws that strike faster than a sword, and almost as hard."
```

Comment:
```suggestion
Among orcs, these riders are treated as a class above regular goblins even when separated from their mounts. Orcish fighters usually feel superior to their lesser kin, but also recognize that picking a fight with the rider will only enrage the beast, whose jaws are as swift and powerful as a sword."
```

Reply:
Could also be "as swift and powerful as any sword".

Reply:
It's a 7-4 attack, compared to the grunt's 9-2. Much swifter, but not quite as much damage.